specification/authorization/resource-manager/readme.csharp.md

```diff
@@ -0,0 +1,30 @@
+# C# Authorization
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for DNS.
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+## Common C# Settings
+
+``` yaml !$(MultiApi)
```

Comment:
@fearthecowboy Just to confirm, `MultiApi` and `multiapi` are the same for Autorest bootstrap? IOW, If I call `autorest --multiapi` does it trigger correctly `MultiApi` here?